crates/holochain_conductor_api/src/config/conductor.rs

```diff
@@ -412,6 +398,32 @@ impl NetworkConfig {
                 serde_json::Value::String(self.signal_url.as_str().into()),
             )?;
 
+            // webrtcConnectTimeout is set to the floor of 1/2 of the request_timeout_s.
+            let timeout_s = serde_json::Number::from_u128(self.request_timeout_s as u128 / 2)
+                .ok_or(ConductorConfigError::InvalidNetworkConfig(
+                    "request_timeout_s must be a number".to_string(),
+                ))?;
+            Self::insert_module_config(
+                module_config,
+                "tx5Transport",
+                "timeoutS",
+                serde_json::Value::Number(timeout_s),
+            )?;
+
+            // webrtcConnectTimeout is set to the floor of 3/8 of the request_timeout_s.
+            let webrtc_connect_timeout = serde_json::Number::from_u128(
```

Comment:
u128? That seems like too many seconds. Even u64 is pretty huge. I think u16 easily covers all the values we'd ever use. It's fine to leave whatever the chosen type is in the existing code, but I think bigger shouldn't be needed?

Reply:
~Yeah, `serde_json::Number` only provides `from_f64`, `from_i128`, and `from_u128` constructors. Strange, maybe a PR to that project is needed.~ Nevermind there are `From` implementations, will fix.